frontend/src/App.tsx

```diff
@@ -18,6 +21,12 @@ class App extends React.Component {
           <Header />
           <main>
             <Switch>
+              <Route path="/login">
+                <CognitoLoginAndReg />
+              </Route>
+              <Route path="/custom-login-reg">
+                <LoginAndRegisteration />
+              </Route>
```

Comment:
Why do we have two login and registration forms?

Reply:
This other login form is the one I created from the mock-ups, but we will not be needing it since Cognito provides a UI for us. I added this route so that it can be viewed.